features/dist-archive.feature

```diff
@@ -224,3 +224,21 @@ Feature: Generate a distribution archive of a project
     And the wp-content/plugins/hello-world/hello-world.php file should exist
     And the wp-content/plugins/hello-world/.travis.yml file should not exist
     And the wp-content/plugins/hello-world/bin directory should not exist
+
+  Scenario: Warns but continues when no distignore file is present
+	Given an empty directory
+	And a test-plugin.php file:
+      """
```

Comment:
Note that you're mixing spaces with tabs here multiple times

Reply:
Sorry, looks like it's spaces I should be using. I'd run phpcbf but hasn't considered the test.